docs/source/API/core/parallel-dispatch/single.md

```diff
@@ -0,0 +1,84 @@
+# `fence()`
+
```

Comment:
why is there a `fence()` call here?

Reply:
If it was intended to be the title it should also contain a label so other pages can reference it.